code/controllers/subsystem/traumas.dm

```diff
@@ -0,0 +1,40 @@
+SUBSYSTEM_DEF(traumas)
+	name = "Traumas"
+	flags = SS_NO_FIRE | SS_NO_INIT
+
+	var/list/phobia_types = list("spiders", "space", "security", "clowns", "greytide", "lizards", "skeletons")
+	var/list/phobia_words = list("spiders"  = list("spider","web","arachnid"),
+								"space"     = list("space", "star", "universe", "void"),
+								"security"  = list(" sec ", "security", "shitcurity", "stunbaton", "taser", "beepsky"),
+								"clowns"    = list("clown", "honk", "banana", "slip"),
+								"greytide"  = list("assistant", "grey", "gasmask", "gas mask", "stunprod", "spear", "revolution", "viva"),
+								"lizards"   = list("lizard", "ligger", "hiss", " wag "),
+								"skeletons" = list("skeleton", "milk", "xylophone", "bone", "calcium", "the ride never ends")
+								)
+
+	var/list/phobia_mobs = list("spiders"  = typecacheof(/mob/living/simple_animal/hostile/poison/giant_spider),
```

Comment:
I'd prefer these lists be created in initialize.

Reply:
i could've sworn i did that, maybe i forgot to push it